package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "emoji-picker-react",
-  "version": "3.4.5",
+  "version": "3.4.6",
```

Comment:
Shouldn't we have a PR for fullstack with updated version of this package?

Reply:
Of course we should, I will open it after this will be merged